hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/autoReloadPanel/autoReloadPanel.tsx

```diff
@@ -49,9 +51,12 @@ class AutoReloadPanel extends React.Component<IAutoReloadPanelProps> {
         >
           {moment(lastRefreshed).format('LT')}
         </Tooltip>
-      );
-
-      const omDBDeltaFullToolTip = <span>
+       );
+    
+     const omSyncStatusDisplay = omStatus === '' ? '' : omStatus ? <div>OM DB update is successfully triggered.</div> : <div>OM DB update is already running.</div>;
```

Comment:
does the second _omStatus_ variable check if the string is true or false?

Reply:
omStatus may contain three values  Empty space when we are not clicking to button and true and false depending upon  END Point Response.

sync trigger endpoint returns false: update button tooltip can show "OM DB update is already running", and becomes gray and unclickable
sync trigger endpoint returns true: update button tooltip can show "OM DB update is successfully triggered", also becomes gray and unclickable

Reply:
ohh i see, thanks!!